source/retryable-writes/tests/insertOne-serverErrors.yml

```diff
@@ -481,4 +481,6 @@ tests:
             result:
                 errorLabelsContain: ["RetryableWriteError"]
             collection:
-                data: []
+                data:
+                    - { _id: 1, x: 11 }
```

Comment:
While you're changing this, can we use `{ _id: 3, x: 33 }` for the `insertOne` argument? That's more consistent with the other tests in this file.

Looking at the full test, I realized that it might fail with a duplicate key exception if `closeConnection` on the fail point wasn't hit for some reason. I'm being paranoid, but that could get weird if a driver's spec runner happened to incorrectly pass because it saw `outcome.error: true` and ignored the `outcome.result.errorLabelsContain` check.

Reply:
Made that change. Regarding your second point I think we should assume that driver's have implemented their test runners appropriately. A duplicate key exception would also be enough of a hint I think that things were amiss, and the test runner should be evaluated.